### PR TITLE
Add all content of capstone project into develop branch

### PR DESCRIPTION
**Pull request by imranhossainemi**
### This project contained different features and works 😄 

The JavaScript capstone project Updates 👍 :

- Add TVmaze API: data about TV series and movies. 🆗 

- Update interfaces 🆗 

- Designing Interactive Interface 😄 
 -   Update the homepage's basic structure with html and css 🆗 
 -   update the comment popup basic structure with html and css 🆗 

- Update home page dom-manipulation 🆗 
 -   add selected API and shows the list of items on the screen 🆗 
-   add Involvement API to show the item likes. 🆗 

- Update comment popup dom manipulation

 - add selected API and shows details about the selected item 🆗 
 - add Involvement API to show the item comments. 🆗 

- Add counters to show interfaces 🆗 

 - add the number of items (home). 🆗 
  - add the number of comments (comments popup). 🆗 
-  Add test for every endpoint 🆗

### CODE REVIEW

File: src/modules/apitest.test.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+
+require('jest-localstorage-mock');
+
+describe('Check all Movies count', () => {
+  it('Movies count be 15', async () => {
+    const newMovies = new Movies();
+    const allMovies = await newMovies.getMovies();
+    const slicedMovie = allMovies.slice(0, 15);
+    expect(slicedMovie.length).toBe(15);
+  });
```
[the remark this answers is not in the document]
Comment (peterrobert) [reply src/modules/apitest.test.js:11]: This change has been done well

File: README.md [hunk 2]
```diff
@@ -29,93 +58,136 @@
...
+ gem install rails
+```
+ -->
+
+### Setup
+
+- Clone this repository to your desired folder:
+  **[https://github.com/byiringiroscar/movies-final-capstone-microverse-javascript ]**
+
+
```
[the remark this answers is not in the document]
Comment (peterrobert) [reply README.md]: This change has been done well

**Review by peterrobert — CHANGES_REQUESTED** [review]
Hi @imranhossainemi and @byiringiroscar  , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but some are still not addressed yet.

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Great work adding the count should be zero tests. However, after running your tests, one of the tests failed. Kindly check below ⬇️ 
test failed

It will be great to ensure that all of your tests are working. Kindly fix this issue and submit your project for another review

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏


Feel free to leave any questions or comments in the PR thread or tag me @peterrobert if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by byiringiroscar** [comment]
### please kindly check again test because, in my local machine, it was working fine 💯 
image

please @peterrobert  check for me again 🙏

**Review by TadesseDev — APPROVED** [review]
### Hi Team

### I'm Tadesse Alemayehu your reviewer for this PR. :hand: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy (1)



## Highlights
 - [x] Good job making the changes requested by the previous code reviewers. ✅
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**